vnext/Microsoft.ReactNative/Utils/TextHitTestUtils.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <UI.Xaml.Controls.h>
+#include <UI.Xaml.Documents.h>
+
+namespace Microsoft::ReactNative {
+struct TextHitTestUtils final {
+  static bool HitTest(const xaml::Documents::Run &run, const winrt::Point &targetPoint);
```

Comment:
there's also a HitTest method in VirtualTextViewManager - can we keep both either in the VM or this Utils file (somewhat prefer the VM: fewer compilation units might help our long build times, and might make understanding this complicated flow easier?

Reply:
They are actually different implementations. The HitTest in VM recurses the shadow node tree. This HitTest is strictly testing whether a Run is pressed.

Reply:
#wontfix